server/routes/api.js

```diff
@@ -7,9 +7,14 @@ const parse = multer();
 // used in excelController.read
 const excelController = require('../controllers/excelController');
 
-router.get('/', (req, res) => {});
-router.post('/', parse.any(), excelController.read);
-router.patch('/', (req, res) => {});
-router.delete('/', (req, res) => {});
+router.get('/', (req, res) => { });
+router.post('/',
+  parse.any(),
+  excelController.read,
+  excelController.convertInputs,
+  excelController.getDataTypes
+);
+router.patch('/', (req, res) => { });
+router.delete('/', (req, res) => { });
```

Comment:
If something is not implemented, we may want to send a message back accordingly so the user (and develop too) knows about it.